ports/aurora-au/vcpkg.json

```
@@ -2,8 +2,10 @@
   "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
   "name": "aurora-au",
   "version-semver": "0.3.5",
+  "port-version": 1,
   "description": "A C++14-compatible physical units library with no dependencies and a single-file delivery option. Emphasis on safety, accessibility, performance, and developer experience.",
   "homepage": "https://github.com/aurora-opensource/au",
+  "documentation": "https://aurora-opensource.github.io/au/main/",
```

Nit: if this is specifically tied to 0.3.5 (line 4), maybe it's best to link to the 0.3.5 version of the docs?

```suggestion
  "documentation": "https://aurora-opensource.github.io/au/0.3.5/",
```

---

A versioned link might be easily overlooked on port updates.

---

Good point.  I'm not familiar with how vcpkg is typically used.  Do people provide only a single version for each library, or multiple versions?  If the former, then the `main` link could be OK.  If the latter, it seems like the version-specific doc link makes more sense.

---

We will provide the latest version to users, and only one version will be provided each time.